src/edu/solvd/mentoring/Garage.java

```diff
@@ -5,25 +5,49 @@
 import java.util.HashMap;
 import java.util.Map;
 
+/**
+ * Garage instance that contains details in stock that can be bough(in future)
+ * stock - a list of details appropriate for sale + price for details
+ */
 public class Garage {
   private static Map<CarDetail, Integer> stock = new HashMap<>();
 
   public static Map<CarDetail, Integer> getStock() {
     return stock;
   }
 
+
+  /**
+   * place given detail with a current price to a stock
+   * @param detail
+   * @param price
+   */
   public static void addCarDetail(CarDetail detail, Integer price){
     stock.put(detail,price);
   }
 
+  /**
+   * removes detail from stock
+   * @param detail - detail to be removed (bought)
+   */
   public static void buyCarDetail(CarDetail detail){
```

Comment:
buy and remove? how?

Reply:
so when we buy smth from the shop it is removed from the shops stock.
 (it will be updated in order to somehow give the chosen detail to player) and then delete from a garage (which si more like a shop)

Reply:
ok